apps/comments/src/views/Comments.vue

```diff
@@ -226,7 +227,7 @@ export default {
 					itemType: 'files',
 					itemId: this.ressourceId,
 					sorter: 'commenters|share-recipients',
-					limit: OC.appConfig?.comments?.maxAutoCompleteResults || 25,
+					limit: loadState('comment', 'maxAutoCompleteResults'),
```

Comment:
You can use a fallback 
```suggestion
					limit: loadState('comment', 'maxAutoCompleteResults', 25),
```

Reply:
So I left it out on purpose... because also in the other case the limit should always be set... (and there teh default is 10). If the limit is not set we probably have bigger issues.

Reply:
This created an issue.
The initial state is `comments` not `comment`.
Comments are broken on master